openfisca_france_local/departements/eure_et_loir/aide-menagere.py

```diff
@@ -0,0 +1,94 @@
+    reference = [
+        "Titre 3 Chapitre 1-1 du Règlement départemental d'Aide Sociale PA/PH de l'Eure-et-Loir",
+        "https://github.com/openfisca/openfisca-france-local/wiki/files/departements/eure-et-loir/RDAS_valide__decembre_2019.pdf"
+    ]
+    documentation = """L’aide-ménagère est accordée aux personnes en situation de handicap ayant besoin, pour demeurer à leur domicile, d’une aide matérielle pour effectuer les actes de la vie courante. 
+                    La participation financière du bénéficiaire est définie par le Conseil départemental.
+                    Cette aide n’est pas cumulable avec l’Allocation compensatrice pour tierce personne (ACTP), la Majoration Tierce Personne (MTP versée par la CPAM) et les prestations d’aide-ménagère servies par les caisses de retraite
+                    """
+
+    def formula_2020_01(individu, period, parameters):
+        taux_incapacite = individu('taux_incapacite', period)
+        restriction_substantielle_durable = individu('aah_restriction_substantielle_durable_acces_emploi', period)
+        age = individu('age', period)
+        ressortissant_eee = individu('ressortissant_eee', period)
+
+        individual_resource_names = {
+            'aah',
+            'salaire_imposable',
+            'retraite_imposable',
+            'pensions_invalidite',
+            'revenus_stage_formation_pro'
+        }
+        ressources_famille = {'aspa'}
+        ressources_annuelles = {'retraite_complementaire_artisan_commercant',
+                                'retraite_complementaire_profession_liberale'
+                                }
+        individu_resources_month = sum(sum([individu(resource, period.last_month) for resource in individual_resource_names]),
+                                 sum([individu.famille(resource, period.last_month) for resource in ressources_famille]))
+        individu_resources = sum(individu_resources_month,sum([individu(resource, period, options = [DIVIDE]) for resource in ressources_annuelles]))
+
+        condition_residence = individu.menage('eure_et_loir_eligibilite_residence', period)
+        parameters_chemin = parameters(
+            period).departements.eure_et_loir.aide_menagere
+
+        condition_taux_incapacite = ((taux_incapacite >= parameters_chemin.taux_incapacite_superieur)
+                                     + ((taux_incapacite < parameters_chemin.taux_incapacite_maximum_restriction_acces_emploi) * ((taux_incapacite > parameters_chemin.taux_incapacite_minimum_restriction_acces_emploi) * restriction_substantielle_durable)))
+        condition_age = (age <= parameters_chemin.age_minimal_personne_handicap)
+        condition_nationalite = ressortissant_eee + individu('refugie',period) + individu('apatride', period)
+        condition_ressources = individu_resources < individu.famille('aspa', period)
+
+        return condition_residence * condition_taux_incapacite * condition_age * condition_nationalite * condition_ressources
```

Comment:
Ajouter les règles de non-cumul ? (RDAS pages 54-en-haut/50-en-bas)

```suggestion
        condition_aides_actp = not_(possede_actp)
        condition_aides_mtp = not_(possede_mtp)
        condition_aide_menagere_caisse_retraite = not_(individu('aide_menagere_fournie_caisse_retraite', period.last_month))
        conditions_non_cumul = condition_aide_menagere_caisse_retraite * condition_aides_actp * condition_aides_mtp
        return condition_residence * condition_taux_incapacite * condition_age * condition_nationalite * conditions_non_cumul * condition_ressources
```

Reply:
Pour PA/PH, nous avons `aide_menagere_fournie_caisse_retraite` d'après le mois précédent. Il me semble que ce pourrait être "pas d'aide ménagère actuellement/au mois courant". As-tu vu la référence au mois précédent dans un règlement ou avec les référents métier ? 🤔

Reply:
Je pense que tu as raison c'est plutôt le mois courant